core/src/main/kotlin/com/malinskiy/marathon/io/FileManager.kt

```diff
@@ -11,9 +11,9 @@ import java.nio.file.Paths.get
 
 @Suppress("TooManyFunctions")
 class FileManager(private val output: File) {
-    fun createFile(fileType: FileType, pool: DevicePoolId, device: DeviceInfo, test: Test): File {
+    fun createFile(fileType: FileType, pool: DevicePoolId, device: DeviceInfo, test: Test, testBatchId: String? = null): File {
```

Comment:
Why do we need a default value for `testBatchId` param?

Reply:
Some output files are not associated with a batch, e.g. test `json`